experimental/envoy_proxy/scripts/common.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC2034  # Unused variables OK as this script is `source`d.
+
+readonly SCRIPTS_DIR="$(dirname "$0")/../../../scripts/"
+# shellcheck source=scripts/common
+source "$SCRIPTS_DIR/common"
+
+readonly ENVOY_CLIENT_IMAGE_NAME='gcr.io/oak-ci/envoy-proxy-client-example'
```

Comment:
Should this (and the next one) be something like this, to clarify the hierarchy?

```suggestion
readonly ENVOY_CLIENT_IMAGE_NAME='gcr.io/oak-ci/envoy-proxy-example-client'
```